cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -241,7 +241,7 @@ img_mgmt_find_by_ver(struct image_version *find, uint8_t *hash)
     int i;
     struct image_version ver;
 
-    for (i = 0; i < 2; i++) {
+    for (i = 0; i < 2*CONFIG_UPDATEABLE_IMAGE_NUMBER; i++) {
```

Comment:
I guess this needs `CONFIG_UPDATEABLE_IMAGE_NUMBER` to be set  if not you'll do 2 * <nothing>